src/rpdk/core/fragment/generator.py

```diff
@@ -75,11 +77,45 @@ def validate_fragments(self):
         self.__validate_no_transforms_present(raw_fragments)
         self.__validate_outputs(raw_fragments)
         self.__validate_mappings(raw_fragments)
+        lint_warnings = self.__validate_fragment_through_cfn_lint(raw_fragments)
+        if not lint_warnings:
+            LOG.warning("Module fragment is valid.")
+        else:
+            LOG.warning(
+                "Module fragment is probably valid, but there are "
+                "warnings/errors from cfn-lint "
+                "(https://github.com/aws-cloudformation/cfn-python-lint):"
+            )
+        for lint_warning in lint_warnings:
+            print(
+                "\t{} (from rule {})".format(lint_warning.message, lint_warning.rule),
+            )
```

Comment:
Would you mind wrapping this in a new method? Something, like print `cfn_lint_warnings`

Reply:
Sorry, I wasn't super clear. I actually meant the entire thing. This way the validate_fragments method will be really short and concise and just have a list of other validate calls. Don't want to block the PR though.

Reply:
refactored!